source/common/config/well_known_names.cc

```diff
@@ -20,16 +20,21 @@ std::string expandRegex(const std::string& regex) {
               // underscores.
               {"<CIPHER>", R"([\w-]+)"},
               // A generic name can contain any character except dots.
-              {"<NAME>", NAME_REGEX},
+              {"<NAME>", TAG_VALUE_REGEX},
```

Comment:
sorry  I forgot about this indirection. For consistency, `s/<NAME>/<TAG_VALUE>/g` .  Thanks!

Reply:
changed